addons/grenades/functions/fnc_throwGrenade.sqf

```diff
@@ -37,9 +37,31 @@ if (local _unit) then {
     };
 
     if (getNumber (_config >> QGVAR(flashbang)) == 1) then {
+        private _amount = 1;
+        if (isNumber (_config >> QGVAR(flashbangBangs))) then {
+            _amount = getNumber (_config >> QGVAR(flashbangBangs));
+        };
+
         private _fuzeTime = getNumber (_config >> "explosionTime");
 
-        [FUNC(flashbangThrownFuze), [_projectile], _fuzeTime] call CBA_fnc_waitAndExecute;
+        private _interval = 0.5;
+        if (isNumber (_config >> QGVAR(flashbangInterval))) then {
+            _interval = getNumber (_config >> QGVAR(flashbangInterval));
+        };
+
+        private _standardDeviation = 0.1;
```

Comment:
```suggestion
        private _intervalSpread = 0.1;
```

Reply:
I'd rather leave this as standardDeviation as it's more precise. I used spread as an imprecise shorthand for standardDeviation.

Reply:
Then we change the config entry to that as well.